tsconfig.base.json

```diff
@@ -14,7 +16,7 @@
     "noImplicitOverride": true,
 
     // We need node's module resolution, so we do not have to skip lib checks
-    "moduleResolution": "Node",
+    "moduleResolution": "Node16",
```

Comment:
Can we also add `"module": "Node16"` here? I don't even know from the top of my head whether it's automatically set. Setting it explicitly seems a bit more clear.

Reply:
Actually it was already there, just above a bit. I'll move it down to be with this though.

Reply:
Ah i see, no i got confused.

Reply:
Anyways, it's here now.